wp-admin/js/widgets/media-image-widget.js

```diff
@@ -39,15 +39,20 @@
 
 			props = {
 				attachment_id: attachment.id,
-				url: attachment.sizes[ displaySettings.size ].url,
-				size: displaySettings.size,
-				width: 0, // Reset.
-				height: 0, // Reset.
 				align: displaySettings.align,
-				caption: attachment.caption,
 				alt: attachment.alt,
+				caption: attachment.caption,
+				image_classes: '',
+				image_title: '',
+				link_classes: '',
+				link_rel: '',
+				link_url: displaySettings.linkUrl,
+				link_target_blank: false,
```

Comment:
@obenland I think that any props for display settings not shown in the select frame should not get reset:

![image](https://cloud.githubusercontent.com/assets/134745/23963313/0f698038-096e-11e7-85f6-2cd3ef7a93c3.png)

So I think that the `align`, `link_type`, `link_url`, and `size` (and `width`, `height`, `url`) should all get reset or overridden from `displaySettings` but everything else can persist across image selections so that the user doesn't have to re-supply them. What do you think?